src/components/BookForm.js

```diff
@@ -17,24 +17,25 @@ export default function BookForm() {
     setCategory(event.target.value);
   };
 
-  const handleSubmit = (event) => {
+  const handleSubmit = async (event) => {
     event.preventDefault();
     const obj = {
       category,
       title: inputValue,
       item_id: uuidv4(),
       progress: 0,
       state: 'Chapter 1',
+      author: 'Suzan Colins',
     };
-    dispatch(addBook(obj));
-    setInputValue('');
+    await postBook(obj);
+    await dispatch(fetchBooks());
   };
 
   return (
     <div className="add-book">
       <span className="hr-line" />
       <p className="heading">ADD NEW BOOK</p>
-      <form onSubmit={(e) => { handleSubmit(e); }}>
+      <form onSubmit={(e) => handleSubmit(e)}>
         <input className="input-title" placeholder="Book title" value={inputValue} onChange={(e) => { handleInput(e); }} />
         <select className="select-category" onChange={(e) => handleSelect(e)}>
           <option className="select-placeholder">Category</option>
```

Comment:
- **[OPTIONAL]** I recommend adding an author input even it is not specified in the wireframe, but it will be useful for the next milestone.